### PR TITLE
switch image uploads to text fields & support avatar

### PR DESCRIPTION
**Pull request by asood123**

### CODE REVIEW

File: frontend/src/containers/AddGroup.js [hunk 1]
```diff
@@ -198,27 +186,37 @@ export class AddGroup extends Component {
                          <Input
                           name='email'
                           value={users1.email}
-                          onChange={(value) => appendGroupForm({users1: {name: users1.name, email: value, twitterHandle: users1.twitterHandle}})}
+                          onChange={(value) => appendGroupForm({users1: {name: users1.name, email: value, twitterHandle: users1.twitterHandle, avatar: users1.avatar}})}
                           maxLength={50}
```
Comment (carlosascari) [inline frontend/src/containers/AddGroup.js]: I've noticed several fields are limited to 50 characters, this *may* one day be an issue. An **email** for example can have up to [254 characters](https://en.wikipedia.org/wiki/Email_address#Syntax)

Are you sure you want to limit it to 50?

**Comment by carlosascari** [comment]
@asood123  LGTM

**Comment by asood123** [comment]
merging this to unblock us.

File: frontend/src/containers/AddGroup.js (hunk 1 above)
Comment (asood123) [inline frontend/src/containers/AddGroup.js]: I suspect that we'll change this flow long before then ;-)

**Comment by xdamman** [comment]
What's the rationale for doing this?

**Pull request merged**